scm_integrations.go

```diff
@@ -0,0 +1,108 @@
+package scalingo
+
+import (
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/go-scalingo/http"
+)
+
+type SCMType string
+
+// Type of SCM integrations
+const (
+	SCMGithubType           SCMType = "github"             // GitHub
+	SCMGithubEnterpriseType SCMType = "github-enterprise"  // GitHub Enterprise (private instance)
+	SCMGitlabType           SCMType = "gitlab"             // GitLab.com
+	SCMGitlabSelfHostedType SCMType = "gitlab-self-hosted" // GitLab self-hosted (private instance)
+)
+
+func (t SCMType) Str() string {
+	return string(t)
+}
+
+type SCMIntegrationsService interface {
+	SCMIntegrationsList() ([]SCMIntegration, error)
+	SCMIntegrationsCreate(scmType SCMType, url string, accessToken string) (*SCMIntegration, error)
+	SCMIntegrationsDestroy(id string) error
+}
+
+var _ SCMIntegrationsService = (*Client)(nil)
+
+type SCMIntegration struct {
+	ID          string  `json:"id,omitempty"`
+	SCMType     SCMType `json:"scm_type"`
+	Url         string  `json:"url"`
+	AccessToken string  `json:"access_token"`
+	Uid         string  `json:"uid,omitempty"`
```

Comment:
Isn't it a UUID?

Reply:
`Uid` is the user id in the SCM integration.

For GitHub for example, `Uid` is the User ID on GitHub, and usually, it's an int.